docs/paper.md

```diff
@@ -94,16 +94,18 @@ In order to meet the needs of researchers, `scores` provides the following key b
 
 **Compatability**
 
-- Highly modular and avoids extensive dependencies by providing its own implementations where relevant.
-- Easy to integrate and use in a wide variety of environments. It has been tested and used on workstations, servers and in high performance computing (supercomputing) environments.
+- Highly modular - provides its own implementations, avoids extensive dependencies and allows greater API consistency.
```

Comment:
Re.: "allows greater API consistency."

I am concerned this phrasing may be confusing to readers who don't already know what the authors intend to say. However, I am also struggling to propose phrasing that might be clearer.

In particular, I am worried it could be misunderstood to mean "allows greater consistency between the `scores` API and other APIs", rather than "allows the `scores` API to be designed with a greater degree of consistency within itself" (or words to that effect, given my wording here is terrible!).

Reply:
I have pushed up a clarification, I thought it would show more clearly here. I have reworded the sentence to "Highly modula - provides its own implementations, avoids extensive dependencies and offers a consistent API. I think this is an improvement, thanks for the feedback.